### PR TITLE
tutorial: Add some precisions in the first lessons

### PR DESCRIPTION
**Pull request by tahini**
Some revisions as per Hubert's comments

### CODE REVIEW

File: tutorial/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ It supposes you have access to an instance of Transition, with access to a routi
 
 [GTFS](https://gtfs.org/) is a open standard format used to distribution transit information. Most agencies in the world use this format, that can be used by various tools to show transit data to users.
 
-Many transit feeds can be easily found in the [Open Mobility data](https://transitfeeds.com/) website, but the most up to date versions are recorded in the [Mobility Database catalogs](https://database.mobilitydata.org/). The transit agencies website may also contain this information. For example, the Montreal Transportation Society, whose data has been used to write this tutorial, can be found on [their website](https://www.stm.info/en/about/developers).
+Many transit feeds can be easily found in the [Open Mobility data](https://transitfeeds.com/) website, but the most up to date versions are recorded in the [Mobility Database catalogs](https://database.mobilitydata.org/). The transit agencies website may also contain this information. For example, the Société de transport de Montréal, the Montreal transit society, whose data has been used to write this tutorial, can be found on [their website](https://www.stm.info/en/about/developers).
```
Comment (kaligrafy) [inline tutorial/README.md]: I would prefer Montreal transit agency instead is society which sound weird in English

**Review by kaligrafy — CHANGES_REQUESTED** [review]

File: tutorial/README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Transition tutorial
 
-This tutorial will guide you through the first steps of using [Transition](http://transition.city) and help you discover what can be achieved with it.
+This tutorial will guide you through the first steps of using [Transition](http://transition.city) and help you discover what can be achieved with it. Transition is a web-based open source platform used for transit planning. You can import and edit transit networks, make various trip calculations, compare scenarios and analyze various metrics of the network.
```
Comment (greenscientist) [inline tutorial/README.md]: Maybe: "You can create, import and edit..." ?

File: tutorial/README.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ If there are errors, the import tab will remain open and display the errors and
 
 *If you know that the GTFS is valid, don't hesitate to open a new issue on [the Transition github](https://github.com/chairemobilite/transition), with a description, the error message and a link the GTFS file used.*
 
-But at this point, there should be data available in Transition!
+But at this point, there should be data available in Transition! You can see what was just imported, navigate the lines and paths, view the stop nodes, etc.
```
Comment (greenscientist) [inline tutorial/README.md]: Do we navigate the line or we navigate the interface and scroll the map and see the various line and path?

**Review by greenscientist — COMMENTED** [review]
Just a few small comment, feel free to ignore if you want.

**Review by kaligrafy — APPROVED** [review]

**Review by greenscientist — APPROVED** [review]

**Pull request merged**